webpack.dev.config.js

```diff
@@ -27,8 +27,12 @@ var cf = {
 			exclude: /node_modules/,
 			loaders: ["raw-loader"]
 		},{
+			test: /\/page1.*\.html$/,
```

Comment:
Ok thank ou for these explanations, I understand :), I will soon remove ngTemplate, and use html-loader as suggested. My main problem stay the problem of templates loaded under Windows OS as mentioned in our issue. I don't know if it's a problem around ui-router / htmlLoader or an other problem. My knowledges are limited around all of this.